production/user_guide.md

```diff
@@ -207,6 +207,58 @@ You will have twenty minutes before your token expires, and you will need to get
 
+
+### Filtering your requests using `_since`
+
+The `_since` parameter lets you filter your bulk data requests by the date when it was updated. This means that instead of receiving all your historical data each time you request data from the API, you will instead be able to enter the date since you last requested data. The API will return data updated between your `_since` input date and the present. The `since` parameter can be used with requests for all resource types.
+
+Before using `_since` for the first time, pull your historical data. Using the `_since` parameter subsequently comprises three steps:
+
+1. Input a date in the correct format.
+2. Start the job to acquire data from an endpoint.
+3. Retrieve data via a job request.
+
+#### a. Pull your historical data.
+
+Before using `_since` for the first time, we recommend that you retrieve all historical data from the BCDA bulk data endpoints (do not use `_since`). Retrieving your historical data before before filtering bulk data with `_since` ensures that there will be no gaps in the claims data delivered from BCDA. 
+
+See [Making Your First Requests for Data](https://bcda.cms.gov/production/user-guide/#making-your-first-requests-for-data) for step-by-step instructions on how to pull your historical data.
+
+After retrieving your historical data, it may be helpful to use the date of your most recent bulk data request for subsequent data pulls using the `_since` parameter. You may retrieve this date by viewing the [_transactionTime_](https://hl7.org/Fhir/uv/bulkdata/export/index.html#response---complete-status) from your last bulk data request.
+
+**Note: Do not input dates before 02-12-2020 into `_since`. Limitations of the Beneficiary FHIR Data (BFD) Server prevent data before 02-12-2020 from being tagged correctly.  For more details, see the [Advanced User Guide](https://bcda.cms.gov/production/technical-user-guide/#filtering-your-data-with-_since).**
+
+#### b. Input a date in the correct format.
+
+<img class="ug-img" src="/assets/img/since_Screenshot_1.png" alt="" />
+<img class="ug-img" src="/assets/img/since_Screenshot_2.png" alt="" />
+
+First, click “Try it Out” in the Swagger section for `_since`. Then, enter your desired date into the dialog box labeled "`_since` (Optional)". Dates and times submitted in `_since` must adhere to a specific format for the server to understand. That format is the FHIR _dateTime_ format (`YYYY-MM-DDThh:mm:ss+zz:zz`). Notice that, if you need to include a time, a timezone must also be specified (`+zz:zz`).
+
+The example below demonstrates how to convert a date/time combination into the FHIR format.
+
+**Date and Time Example**
+
+* _Sample Date:_ February 20, 2020 12:00 PM EST
+* _dateTime Format:_ YYYY-MM-DDThh:mm:ss+zz:zz
+* _Formatted Sample:_ 2020-02-20T12:00:00.00-05:00
+
+More information about the FHIR dateTime format can be found on the [FHIR Datatypes page](https://www.hl7.org/fhir/datatypes.html#dateTime).
```

Comment:
Can you apply a `{:target="_blank"}` to this link so it opens in a new tab (since it is not related to the `cms.gov` domain)?  There are other examples in this file that you can use as examples.